pkg/registry/core/service/storage/rest_test.go

```diff
@@ -893,52 +679,24 @@ func TestServiceRegistryUpdate(t *testing.T) {
 	if updatedService.Name != "foo" {
 		t.Errorf("Expected foo, but got %v", updatedService.Name)
 	}
-	if e, a := "foo", registry.UpdatedID; e != a {
-		t.Errorf("Expected %v, but got %v", e, a)
-	}
 }
 
 func TestServiceRegistryUpdateDryRun(t *testing.T) {
```

Comment:
if we are going for `exemplar` (your words not mine :-))

I really dislike how this test is written. This is not your PR thing. It is a general statement for these kinds of tests we should we do something like
```
base: MakeService(...) 
steps := []struct{
name: string,
    tweeker: servicesTesting.Tweek,
    checker: func(t *testing.T, original, updated *api.Service),
  } 
original := // create logic
for _, step := range steps{
t.Run(step.Name, func(t *testing.T){
 step.tweeker(&original)
updated := // updated logic
step.checker(t, original, updated)
})
// step might not be the best name. but the general idea remain.
```
}

Reply:
I agree, but let's make the single-layer tests (in storage_test) the good stuff.  Rewriting this now is counter-productive.  FWIW I agree